addons/mass_mailing_themes/views/mass_mailing_themes_templates.xml

```diff
@@ -866,4 +881,750 @@
+            <div class="container o_mail_table_styles">
+                <div class="row">
+                    <div class="col-lg-12 o_mail_footer_social">
+                        <a aria-label="Facebook" title="Facebook" target="_blank" href="https://www.facebook.com/Odoo"><span class="fa fa-facebook text-o-color-4"></span></a> &amp;nbsp;&amp;nbsp;
+                        <a style="margin-left:10px" aria-label="Twitter" target="_blank" title="Twitter" href="https://twitter.com/Odoo"><span class="fa fa-twitter text-o-color-4"></span></a>  &amp;nbsp;&amp;nbsp;
+                        <a style="margin-left:10px" aria-label="Instagram" title="Instagram" target="_blank" href="https://www.instagram.com/explore/tags/odoo/"><span class="fa fa-instagram text-o-color-4"></span></a>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-lg-12 o_mail_footer_links">
+                        <a role="button" href="/unsubscribe_from_list" class="btn btn-link text-o-color-4">Unsubscribe</a> | <a role="button" href="/contactus" class="btn btn-link text-o-color-4">Contact</a>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-lg-12">
+                        <p class="o_mail_footer_copy text-o-color-4">
+                            © <t t-esc="datetime.datetime.now().year"/> All Rights Reserved
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </template>
+
+    <!-- Theme "textnews" default template -->
+    <template id="theme_textnews_template">
+        <style id="design-element">
+            h1 {
+                color: #CE0000;
+                font-size: 54px;
+            }
+            h2{
+                font-weight: bolder;
+            }
+            h3 {
+                font-size: 14px;
+                font-weight: bolder;
+            }
+            p, p > *, li, li > * {
+                font-size: 14px;
+            }
+            a:not(.btn), a.btn.btn-link {
+                text-decoration-line: underline;
+                color: #212529;
+            }
+            a.btn.btn-primary, a.btn.btn-outline-primary, a.btn.btn-fill-primary {
+                background-color: #CE0000;
+                border-color: #CE0000;
+            }
+        </style>
+        <t t-set="now" t-value="datetime.datetime.now()"/>
+        <div class="o_snippet_view_in_browser o_mail_snippet_general pt16 pb0" style="text-align: left; padding-left: 15px; padding-right: 15px; color: rgb(183, 191, 199) !important" data-snippet="s_mail_block_header_view" data-name="View Online"><a href="/view" class="" target="_blank" data-bs-original-title="" title="">View Online</a> - <span style="color: rgb(183, 191, 199); font-size: 12.25px">August 29th 2022</span></div>
```

Comment:
Same here, do we really want to hardcode the date ? It's fine by me if that's what we want but it looks like a mistake.

Reply:
Date is now dynamic